app/helpers/application_helper.rb

```diff
@@ -2,4 +2,12 @@ module ApplicationHelper
   def last_updated_date
     File.mtime(Rails.root.join('REVISION')).to_date rescue Date.today
   end
+
+  def ajax_enabled_for?(name)
+    %w(
+      energy-grants-calculator
+      calculate-employee-redundancy-pay
+      register-a-death
+    ).exclude?(name)
```

Comment:
Building this list to me is brittle.. an ideal solution will for the `Flow` object to provide is ajax enabled capability. 

This way we avoid having to building this list and avoid future increase in elements of the array here.

so just like `@presenter` provides the name of the `Flow`.

It ideal for a `Flow` ajax status to be set in the `define` method.
For example

```ruby
  content_id "yuewyeuw-ueiwueiw-uier76"
  name 'energy-grants-calculator'
  status :published
  satisfies_need "100982"
  ajax_enabled false      
```

And then in the presenter

```ruby
 def ajax_enabled
    @flow.ajax_enabled
 end
```

```javascript
<script>
    var SmartAnswer = SmartAnswer || {};
    SmartAnswer.AJAX_ENABLED = <%= @presenter.ajax_enabled %>;
  </script>
```

@selfthinker This is an ideal scenario